pkg/model/model.go

```diff
@@ -92,6 +92,16 @@ func (p *Plugin) Clone() *Plugin {
 	}
 }
 
+type PluginProxyRouteMap map[string]*PluginRoute
+
+func (pprm *PluginProxyRouteMap) String() string {
+	b, err := json.Marshal(pprm)
+	if err != nil {
+		return err.Error()
```

Comment:
如果这个报错，外层返回的依然是字符串不知道使用的字符串是正确符合预期的还是错误的。

Reply:
结构体的 String 方法在这里主要用于对日志的输出时格式化使用，改成了下面的格式用来突出错误。
```suggestion
		return "<" + err.Error() + ">"
```